airflow/cli/commands/connection_command.py

```diff
@@ -253,7 +254,6 @@ def _import_helper(file_path):
             if session.query(Connection).filter(Connection.conn_id == conn_id).first():
                 print(f'Could not import connection {conn_id}: connection already exists.')
                 continue
-
             session.add(conn)
-            session.commit()
-            print(f'Imported connection {conn_id}')
+            print(f'Load connection {conn_id}')
+    print("Successfully imported all connections.")
```

Comment:
Hmm, I think the previous code is written specifically so if one of the connections fail to import, other previously read connections are still committed to the database (instead of the entire session being rolled back and not committing anything). But I’m not entirely sure which behaviour is more expected by users.

Reply:
-_-| , i will be more careful...